ci-operator/jobs/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main-presubmits.yaml

```diff
@@ -455,13 +445,25 @@ presubmits:
           readOnly: true
       serviceAccountName: ci-operator
       volumes:
+      - name: boskos
+        secret:
+          items:
+          - key: credentials
+            path: credentials
+          secretName: boskos-credentials
+      - name: ci-pull-credentials
+        secret:
+          secretName: ci-pull-credentials
+      - name: cluster-profile
+        secret:
+          secretName: cluster-secrets-gcp-qe
       - name: pull-secret
         secret:
           secretName: registry-pull-credentials
       - name: result-aggregator
         secret:
           secretName: result-aggregator
-    trigger: (?m)^/test( | .* )images,?($|\s.*)
+    trigger: (?m)^/test( | .* )gcp-4.13-nightly-ocp-qe-perfscale-router-perf-gcp,?($|\s.*)
```

Comment:
@vishnuchalla can you re trigger the make jobs. This shouldn't reference a specific test like this I dont think

Reply:
Even running make jobs did not change anything. I think that is just how `prowgen` works.

Reply:
ok! since it has always_run as false think we are ok. I've had issues with tests not "images" type blocking PR's before in ocp-qe-perfscale-ci